packages/debug-adapter/src/debug-session.ts

```diff
@@ -30,56 +31,41 @@ export class WollokDebugSession extends DebugSession {
     this.notifyConfigurationDone()
   }
 
-  protected initializeRequest(response: DebugProtocol.InitializeResponse, _args: DebugProtocol.InitializeRequestArguments): void {
+  protected initializeRequest(response: DebugProtocol.InitializeResponse, args: DebugProtocol.InitializeRequestArguments): void {
     // capabilities
     response.body = {
       ...response.body,
-      // ToDo: supportsBreakpointLocationsRequest: true
+      supportsBreakpointLocationsRequest: true,
       supportsDelayedStackTraceLoading: true,
       supportsConfigurationDoneRequest: true,
       supportsSingleThreadExecutionRequests: false,
     }
 
+    this.positionConverter = new WollokPositionConverter(args.linesStartAt1, args.columnsStartAt1)
+
     // initialize wollok interpreter
     const debuggableFileExtensions = [WOLLOK_FILE_EXTENSION, PROGRAM_FILE_EXTENSION, TEST_FILE_EXTENSION]
     this.workspace.findFiles(`**/*.{${debuggableFileExtensions.join(',')}}`).then(async files => {
       const wollokPackages = await Promise.all(files.map(file =>
 
         new Promise<FileContent>(resolve => this.workspace.openTextDocument(file).then(textDocument => {
-          resolve({ name: this.toWollokPath(textDocument.uri.fsPath), content: textDocument.getText() })
+          resolve({ name: toWollokPath(textDocument.uri.fsPath), content: textDocument.getText() })
         }))
       ))
 
-      this.environment = buildEnvironment(wollokPackages, undefined)
-      this.interpreter = executionFor(this.environment)
+      const environment = buildEnvironment(wollokPackages, undefined)
+      this.interpreter = executionFor(environment)
```

Comment:
Acá el `undefined` es innecesario. Además yo cambiaría `wollokPackages` por `projectFiles`

```suggestion
      const environment = buildEnvironment(projectFiles)
      this.interpreter = executionFor(environment)
```